src/main/java/com/google/devtools/build/lib/skyframe/ConfiguredTargetFunction.java

```diff
@@ -485,11 +485,11 @@ public SkyValue compute(SkyKey key, Environment env) throws ConfiguredTargetFunc
             configuration.getFragmentsMap().keySet(),
             BuildOptions.diffForReconstruction(defaultBuildOptions, toolchainOptions));
 
-    Map<String, UnloadedToolchainContextKey> unloadedToolchainContextKeys = new HashMap<>();
+    Map<String, ToolchainContextKey> unloadedToolchainContextKeys = new HashMap<>();
```

Comment:
Should these relevant var names be changed to, e.g., toolchainContextKeys? Does UnloadedToolchainContext still exist? Sorry if I'm missing context!

Reply:
UnloadedToolchainContext still exists, but yes, these should be renamed.